mkdocs.yml

```diff
@@ -1,20 +1,95 @@
+  - Course management:
+    - 'Student Enrollments': 'features/coursemanagement/studentenrollments.md'
+    - 'Permissions': 'features/coursemanagement/permissions.md'
+    - 'Pinboard abuse reports': 'features/coursemanagement/pinboardabusereports.md'
+    - 'Student Submissions': 'features/coursemanagement/studentsubmissions.md'
+  - Analytics:
+    - Dashboards:
+      - 'Item Visits': 'features/analytics/dashboards/itemvisits.md'
+      - 'Videos': 'features/analytics/dashboards/videos.md'
+      - 'Downloads': 'features/analytics/dashboards/downloads.md'
+      - 'Quizzes': 'features/analytics/dashboards/quizzes.md'
+      - 'Item Details': 'features/analytics/dashboards/itemdetails.md'
+      - 'Section Conversions': 'features/analytics/dashboards/sectionconversions.md'
+      - 'Forum': 'features/analytics/dashboards/forum.md'
+      - 'Activity': 'features/analytics/dashboards/activity.md'
+      - 'Geographic': 'features/analytics/dashboards/geographic.md'
+      - 'Announcements': 'features/analytics/dashboards/announcements.md'
+      - 'QC Alerts': 'features/analytics/dashboards/qcalerts.md'
+      - 'Social Shares': 'features/analytics/dashboards/socialshares.md'
+      - 'Referrer': 'features/analytics/dashboards/referrer.md'
+    - Progress: 'features/analytics/progress.md'
+  - Certificates: 'features/certificates.md'
+  - Learning Objectives: 'features/learningobjectives.md'
+  - Course Announcements: 'features/courseannouncements.md'
+  - External Tools: 
+    - CodeOcean: 'features/externaltools/codeocean.md'
+    - TeamBuilder: 'features/externaltools/teambuilder.md'
+- Best Practices:
+  - 'Getting Started: Design your Course': 'bestpractices/gettingstarted.md'
+  - 'Curriculum Design': 'bestpractices/curriculumdesign.md'
+  - 'Recommended Course Cycles':
+    - Course Cycles: 'bestpractices/recommendedcoursecycles/coursecycles.md'
+    - 'Scheduled': 
+      - Lecture: 'bestpractices/recommendedcoursecycles/scheduled/lecture.md'
+      - Hands-on: 'bestpractices/recommendedcoursecycles/scheduled/handson.md'
+      - Workshop: 'bestpractices/recommendedcoursecycles/scheduled/workshop.md'
+      - Nutshell: 'bestpractices/recommendedcoursecycles/scheduled/nutshell.md'
+    - 'Self-paced': 'bestpractices/recommendedcoursecycles/selfpaced.md'
+    - 'Blended': 'bestpractices/recommendedcoursecycles/blended.md'
+  - Prepare Material: 'bestpractices/preparematerial.md'
+  - Quiz Design: 'bestpractices/quizdesign.md'
+  - Community Management: 'bestpractices/communitymanagement.md'
+  - End of Course: 'bestpractices/endofcourse.md'
+  - Checklist: 'bestpractices/checklist.md'
+  - Project Management Gantt Chart: 'bestpractices/projectmanagement.md'
+- Course Administration:
+  - Setup Course Properties: 'courseadministration/courseproperties.md'
+  - Grant Permissions: 'courseadministration/permissions.md'
+  - Add Content: 
+    - Sections: 'courseadministration/addcontent/sections.md'
+    - Items: 
```

Comment:
The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - courseadministration/addcontent/items/item.md

Reply:
Link to Logo is fixed, path to items.md in mkdocs.yaml is added